NitroxClient/Communication/Packets/Processors/DroppedItemPacketProcessor.cs

```diff
@@ -0,0 +1,63 @@
+﻿using NitroxClient.Communication.Packets.Processors.Abstract;
+using NitroxClient.GameLogic.Helper;
+using NitroxClient.GameLogic.ItemDropActions;
+using NitroxClient.MonoBehaviours;
+using NitroxClient.Unity.Helper;
+using NitroxModel.Logger;
+using NitroxModel.Packets;
+using UnityEngine;
+
+namespace NitroxClient.Communication.Packets.Processors
+{
+    class DroppedItemPacketProcessor : ClientPacketProcessor<DroppedItemPacket>
```

Comment:
the chanthe diference is the data container

 DroppedItem : DeferrablePacket

DroppedItemPacket : Packet

Reply:
some item container i no need be DeferrablePacket

Reply:
Ok that makes sense.  However, I think we should still only have one DropItem packet class.  Since there are very few DeferrablePackets let's remove the DeferrablePacket class and make a virtual function called IsDeferrable() in the Packet base class with a default implementation of "return false".  You can then override this in the DropItem class to conditionally be deferrable or not.  You'll need to modify DeferringPacketReceiver.cs to call this new function instead of using reflection.